internal/cmd/kafka/command_link_configuration_list.go

```diff
@@ -39,43 +37,40 @@ func (c *linkCommand) configurationList(cmd *cobra.Command, args []string) error
 	linkName := args[0]
 
 	kafkaREST, err := c.GetKafkaREST()
-	if kafkaREST == nil {
-		if err != nil {
-			return err
-		}
-		return errors.New(errors.RestProxyNotAvailableMsg)
+	if err != nil {
+		return err
 	}
 
 	cluster, err := c.Context.GetKafkaClusterForCommand()
 	if err != nil {
 		return err
 	}
 
-	listLinkConfigsRespData, httpResp, err := kafkaREST.CloudClient.ListKafkaLinkConfigs(cluster.ID, linkName)
+	configs, err := kafkaREST.CloudClient.ListKafkaLinkConfigs(cluster.ID, linkName)
 	if err != nil {
-		return kafkarest.NewError(kafkaREST.CloudClient.GetUrl(), err, httpResp)
+		return err
 	}
 
 	list := output.NewList(cmd)
-	if len(listLinkConfigsRespData.Data) == 0 {
+	if len(configs.GetData()) == 0 {
 		return list.Print()
 	}
 
 	list.Add(&linkConfigurationOut{
 		ConfigName:  "dest.cluster.id",
-		ConfigValue: listLinkConfigsRespData.Data[0].ClusterId,
+		ConfigValue: configs.GetData()[0].GetClusterId(),
```

Comment:
This seems a bit weird, and forces us to add the special case on line 55. Can we just use `cluster.ID` here?

Reply:
```suggestion
		ConfigValue: cluster.ID,
```